### PR TITLE
React package readme edit

### PR DESCRIPTION
**Pull request by grafsean**
@codecapitano I've made a few minor formatting edits.

How do these sections "### React Router without Data Routers", "### React Router with Data Routers" differ from "### Router"?

Could we have call to go over document structure and revisit where sections are nested?

### CODE REVIEW

File: packages/react/README.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
+- **Error Boundary**: provides additional stack trace for errors and configuration options for pushError behavior
+- **Component Profiler**: capture every re-render of a component, the un/mounting time etc.
+- **React Router (v4 - v6) integration**: send events for all route changes
+- **SSR support**
 
-## Installation
+## React router with data router
 
-### React Router without Data Routers
+To set up Faro-React with React Router V6 without Data routers
```
Comment (grafsean) [inline packages/react/README.md:16]:
```suggestion
To set up Faro-React with React Router V6 with Data routers
```

File: packages/react/README.md [hunk 2]
```diff
@@ -35,37 +38,46 @@ initializeFaro({
...
+const reactBrowserRouter = createBrowserRouter([
+  //...
+]);
 
-import { getWebInstrumentations, initializeFaro, ReactIntegration, ReactRouterVersion } from '@grafana/faro-react';
-import { TracingInstrumentation } from '@grafana/faro-web-tracing';
+const browserRouter = withFaroRouterInstrumentation(reactBrowserRouter);
+```
+
+_upgrading from v6 to v6 data router_
```
Comment (grafsean) [inline packages/react/README.md:68]:
```suggestion
_upgrading from v6 without a data router to v6 with a data router_
```

File: packages/react/README.md [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
-- Component Profiler - Capture every re-render of a component, the un/mounting time etc.
-- Router (v4-v6) integration - Send events for all route changes
-- SSR support
+- **Error Boundary**: provides additional stack trace for errors and configuration options for pushError behavior
+- **Component Profiler**: capture every re-render of a component, the un/mounting time etc.
+- **React Router (v4 - v6) integration**: send events for all route changes
+- **SSR support**
 
-## Installation
+## React router with data router
```
Comment (grafsean) [inline packages/react/README.md:14]: What versions of the router does this section cover?

File: packages/react/README.md [hunk 4]
```diff
@@ -78,68 +90,78 @@ initializeFaro({
...
-
-or
-
-```tsx
-import { withErrorBoundary } from '@grafana/faro-react';
+## React router without
 
-export default withErrorBoundary(App);
-```
+To set up Faro-React with React Router V5 or V6 without Data routers, add the following code snippet
```
Comment (grafsean) [inline packages/react/README.md:126]: There use to be a v4, is that no longer relevant?

File: packages/react/README.md (hunk 2 above)
Comment (codecapitano) [reply packages/react/README.md:68]: Q: What would be the best format for the description?
Heading, or bold or keep it italic or simply no formatting at all?

File: packages/react/README.md (hunk 4 above)
Comment (codecapitano) [reply packages/react/README.md:126]: Good catch tynak you. Forgot to add it.

**Comment by codecapitano** [comment]
@grafsean is this PR ready to merge from your side?

If yes, I can fix the linter issues, merge this PR and then we can copy the content to the fe-o11y docs as well.
I can open a PR on the website if you like.

**Comment by grafsean** [comment]
Hi @codecapitano I am adding more copy today and will have follow up questions. After that I think we will be good to merge.

**Comment by grafsean** [comment]
@codecapitano I added more detail and a few questions. Could we call tomorrow morning to discuss next steps?

**Comment by codecapitano** [comment]
> I added more detail and a few questions. Could we call tomorrow morning to discuss next steps?
Sure and thank you Sean 🙏

File: packages/react/README.md [hunk 5]
```diff
@@ -1,25 +1,26 @@
 # Set up the Faro React distribution of the Faro Web SDK
 
-Faro-React is a distribution of the Faro Web SDK for project using React, which offers easier integrations and the following features:
+The Faro Web-SDK is a highly configurable open source real user monitoring (RUM) library built on OpenTelemetry and integrating seamlessly with Grafana Cloud and Grafana Frontend Observability. Faro-React is a distribution of the Faro Web SDK for project using React, which offers easier integrations and the following features:
```
Comment (codecapitano) [inline packages/react/README.md:3]: @grafsean 
Faro is not built on OpenTelemetry. 
It uses it's own protocol and has nothing in common with  openteletry.

Only exception is the `web-tracing` package which is a wrapper around otel which makes it easier to setup otel.js and to use it with Faro.

Comment (grafsean) [reply packages/react/README.md]: Is the use of OTel through web-tracting automatic and not optional?

Comment (codecapitano) [reply packages/react/README.md]: It's automatic.
The web-tracing package is a wrapper around otel.js, so opentelemetry is not optional for that specific package.
It's meant to make including otel easier.

**Pull request closed without merging**